modules/ip/api/ip-routes.ts

```diff
@@ -0,0 +1,23 @@
+import { Request, Response } from 'express';
+import * as core from 'express-serve-static-core';
+
+function genMAC(){
```

Comment:
Can you move this function into a file under  a util folder in the module

Reply:
moved the function to util folder of respective module